### DISCUSSION TITLE
What is the best cheapest platform to deploy a multi-tenant django app to production

### DISCUSSION THREAD

**Discussion by iamunadike** [django-tenants/django-tenants#773] [Q&A]
@tomturner 
@alexandernst 
I wish to ask you what is the cheapest platform to deploy my django multi-tenant app to production
especially using docker, already I have it setup ,
Digital ocean has a managed database with starter package @ $15/month coupled with the droplet server $6/month, these are the lowest prices, but I am not sure if  I can have just the droplet without the managed database, having postgresql inside the docker,
My question is do you any other platform you use that is cheaper and very effective?
If not how to do this on Digital ocean?
Thanks and your replies is duely appreciated

**Comment by tomturner** [comment]
Personally I would use AWS

**Reply by iamunadike** [reply]
okay but what are the cost monthly and what approach to AWS. Please

**Reply by iamunadike** [reply]
I am just looking for the cheapest possible way out, so I can maintain both short and long term costs, giving me an overiview will come in very handy .Thanks

**Comment by tomturner** [comment]
I would just spin a machine up. Personally not sure of cost as it’s hard to
work the cost out. I would get the system to work in docker then you can
deboy to anywhere.

**Reply by iamunadike** [reply]
Is this running in production , i have a app, which offcourse I will create subdomains on say namecheap and make them point to their schemas, on the api
I really like to know if the above works to physically represent the schemas per subdomain, then how to engine: postgresql_backend engine for django-tenants still works , then having an app that runs to handle the increasing schemas in the future , its quite new for now. 

I think I will like tutorial on how to deploy django-tenants with django apps on production on the AWS using docker and github actions and how to configure further resources to point to schemas, based on the domain , Your help is duely appreciated, Thanks

**Reply by iamunadike** [reply]
but so far are you aware of costs runing your multinenant app on AWS, is there a free Postgresql database and the schemas you have is there a database row limit 
Is it a Free tier ? How can i get into it,send links ,articles etc thanks

**Reply by iamunadike** [reply]
@tomturner @valberg @suriya
Please I need a clarification here:
-- I am a single developer(for now) working on my project which I intend to deploy , but my concerns are deploying it to AWS ec2, what I intend to ask is this , using Docker I hear eats a lots of CPU and memory, I really dont know how safe, for me I will prefer a docker approach and github actions to automate as docker itself, automates running services with one click, but my concerns are is there a risk, will docker slow down the VM, is docker reliable, etc

**Comment by maxwellamaral** [comment]
I like Contabo.com's services. For me, the sole developer, it's the best value for money.
